provisions/roles/cleanup/jenkins/slave/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Remove all containers
+  sudo: yes
+  shell: docker rm -f `docker ps -a -q`
```

Comment:
removing containers would not matter much, because provision will bring all the containers back. Though we have to find a to validate the current  jobs is processed before stopping container.